airflow/jobs.py

```diff
@@ -459,39 +700,24 @@ def schedule_dag(self, dag):
                 session.commit()
                 return next_run
 
-    def process_dag(self, dag, queue):
+    def create_task_instances(self, dag, queue):
```

Comment:
this should probably be called process_task_instances, task_instances are eagerly created thus the name does not reflect what is happening here